weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/CellRenderers.tsx

```diff
@@ -63,7 +63,9 @@ export const CellViewingRenderer: React.FC<
   const [isHovered, setIsHovered] = useState(false);
   const {setEditedRows} = useDatasetEditContext();
 
-  const isEditable = typeof value !== 'object' && typeof value !== 'boolean';
+  const isWeaveUrl = typeof value === 'string' && value.startsWith('weave:///');
```

Comment:
you probably want to use `isRefPrefixedString` OR at the very least use the constants for prefix matching.